extensions/amp-base-carousel/1.0/arrow.js

```diff
@@ -34,16 +33,17 @@ export function Arrow({customArrow, by, advance, disabled}) {
     }
     advance(by);
   };
+  const styles = useStyles();
+  let classNames =
```

Comment:
Nit, but this might be better:

```
const classNames = `${styles.arrowPlacement} ${by < 0 ? styles.arrowLeft : styles.arrowRight} ${isDisabled ? styles.arrowDisabled : ''}`;
```

Just looking for good patterns here since we'll be doing this quite a bit.

Reply:
Done.

I'm in favor of a `classnames` like helper lib going forward. Would transform this to the much clearer:
```js
const  classNames = cx(
    styles.arrowPlacement,
    by < 0 ? styles.arrowLeft : styles.arrowRight,
    {[styles.arrowDisabled]: isDisabled}
)
```